tests/lib/rules/jsx-no-bind.js

```diff
@@ -115,6 +116,121 @@ ruleTester.run('jsx-no-bind', rule, {
         };
       `,
       parser: 'babel-eslint'
+    },
+
+    {
+      code: [
+        'class Hello extends Component {',
+        '  render() {',
+        '    const click = this.onTap.bind(this);',
+        '    return <div onClick={onClick}>Hello</div>;',
+        '  }',
+        '};'
+      ].join('\n'),
+      parser: 'babel-eslint'
+    },
+    {
+      code: [
+        'class Hello extends Component {',
+        '  render() {',
+        '    foo.onClick = this.onTap.bind(this);',
+        '    return <div onClick={onClick}>Hello</div>;',
+        '  }',
+        '};'
+      ].join('\n'),
+      parser: 'babel-eslint'
+    },
+    {
+      code: [
+        'class Hello extends Component {',
+        '  render() {',
+        '    return (<div>{',
+        '      this.props.list.map(this.wrap.bind(this, "span"))',
+        '    }</div>);',
+        '  }',
+        '};'
+      ].join('\n'),
+      parser: 'babel-eslint'
```

Comment:
most of these test cases don't need `babel-eslint`; can we use the default parser wherever possible?